### PR TITLE
Readme & github files

### PR DESCRIPTION
**Pull request by lock9**
New readme, issue templates and contributing.md

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to detail an error or unexpected behavior
+title: ''
+labels: ''
```
Comment (shargon) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Why this table?

Comment (lock9) [reply .github/ISSUE_TEMPLATE/bug_report.md]: It is a GitHub configuration for the issue templates (button title and description)

File: .github/ISSUE_TEMPLATE/feature-or-enhancement-request.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+name: Feature or enhancement request
+about: Suggest an idea for Neo
+title: ''
+labels: discussion
+assignees: ''
+---
+
+**Summary or problem description**
+A summary of the problem you want to solve or metric you want to improve
```
Comment (shargon) [inline .github/ISSUE_TEMPLATE/feature-or-enhancement-request.md]: between the title and the content there are no new lines, or is the preview?

File: README.md [hunk 3]
```diff
@@ -1,91 +1,220 @@
...
+<h3 align="center">Neo Blockchain</h3>
+
+<p align="center">
+   A modern distributed network for the Smart Economy.
+  <br>
+  <a href="https://docs.neo.org/docs/en-us/index.html"><strong>Documentation »</strong></a>
+  <br>
+  <br>
+  <a href="https://github.com/neo-project/neo">Neo<</a>
+  ·
```
Comment (shargon) [inline README.md]: remove empty lines in the html

Comment (shargon) [reply README.md]: Ping @lock9

Comment (Tommo-L) [reply README.md:18]: Fixed

File: .github/ISSUE_TEMPLATE/feature-or-enhancement-request.md (hunk 2 above)
Comment (Tommo-L) [reply .github/ISSUE_TEMPLATE/feature-or-enhancement-request.md:10]: It's the preview, don't need to add a new line.

**Comment by superboyiii** [comment]
@shargon @erikzhang Please have a review.

**Comment by shargon** [comment]
Merge?

**Comment by erikzhang** [comment]
Does this modification fit in this repo? Maybe it's better to move to neo-project/neo.

**Comment by Liaojinghui** [comment]
> Does this modification fit in this repo? Maybe it's better to move to neo-project/neo.

Agree, since `core` is the most important project of neo, these high level content should be part of the `core` instead of a `subproject`.

**Pull request closed without merging**